installer/build.rs

```diff
@@ -0,0 +1,17 @@
+fn main() {
+    let mut envs: std::collections::HashMap<String, String> = std::env::vars().collect();
+    let rustflags = envs.entry("RUSTFLAGS".to_string()).or_default();
+    *rustflags = format!("{} -Ctarget-feature=+aes,+ssse3", rustflags);
+    std::process::Command::new("cargo")
+        .args(&[
+            "install",
+            "--force",
+            "--git",
+            "https://github.com/oasislabs/oasis-chain",
+            "--tag",
+            "v0.1.0",
```

Comment:
```suggestion
            "v0.1.0",
            "--debug",
```

Reply:
why `--debug`?

Reply:
Printing stdout/stdout in parity is behind `debug_assertions` flag. Perhaps we just change that to a feature flag.